database/gdb/gdb_model_soft_time.go

```diff
@@ -214,7 +219,7 @@ func (m *softTimeMaintainer) getSoftFieldNameAndType(
 			return
 		}
 	)
-	result, err := m.db.GetCache().GetOrSetFunc(ctx, cacheKey, cacheFunc, cacheDuration)
+	result, err := softFieldNameAndTypeCache.GetOrSetFunc(ctx, cacheKey, cacheFunc, cacheDuration)
```

Comment:
Is t better using `gcache` here instead of new cache object?

Reply:
Thanks for the review. Sure, `gcache` use the default in-memory cache and it fits here.